src/gui/loginrequireddialog/oauthloginrequiredwidget.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+* Copyright (C) Fabian Müller <fmueller@owncloud.com>
+*
+* This program is free software; you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation; either version 2 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful, but
+* WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+* or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+* for more details.
+*/
+
+#include "oauthloginrequiredwidget.h"
+#include "gui/application.h"
+#include "gui/creds/httpcredentialsgui.h"
+#include "gui/guiutility.h"
+#include "ui_oauthloginrequiredwidget.h"
```

Comment:
I know that its nitpicking at that level but please move the ui include below oauthloginrequiredwidget and then add an empty line.